lib/workflowmgr/slurmbatchsystem.rb

```diff
@@ -14,6 +14,9 @@ module WorkflowMgr
   ##########################################
   class SLURMBatchSystem < BatchSystem
 
+    # Need at least this version of SLURM to get heterogeneous job support:
+    HETEROGENEOUS_JOB_VERSION="18.8"
```

Comment:
Please change this to "18.08".  Even though "18.8" is functionally the same, I want the string to be the same as what Slurm actually uses in their versioning.